drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
If handling a function type means we completely change the syntax, and we end up changing it to new syntax which is just as simple to write and parse as the proposed syntax here (say, using a different separator from a comma), shouldn't we just start with the new syntax?